content/22.vaccines.md

```diff
@@ -405,6 +405,116 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
+This was followed by an EUA for the Moderna vaccine on December 18th, 2020 [@doi:10.15585/mmwr.mm695152e1]. 
+On December 20th, 2020, ACIP updated their initial recommendations (phase 1b) suggesting vaccinations should be offered to people aged 75 years old and older and to non-healthcare frontline workers [@doi:10.15585/mmwr.mm695152e2]. 
+On the same date, it was recommended that phase 1c would include people aged 65-74 years old, individuals between the ages of 16-74 years old who are at high-risk due to health conditions, and essential workers that were not eligible in phase 1b [@doi:10.15585/mmwr.mm695152e2].
+On the following day, December 21st, 2020, the first Moderna vaccines used outside of clinical trials were administered to American healthcare workers, which was the same day that President-elect Biden and Dr Biden received their first doses of the Pfizer-BioNTech vaccine live on television to instill confidence in the approval and vaccination process [@url:https://www.cnn.com/2020/12/21/health/us-coronavirus-monday/index.html]. 
+
+On February 27th, 2020, the FDA issued an EUA for the Janssen COVID-19 Vaccine [@url:https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/janssen-covid-19-vaccine]. 
+This was followed by an update on recommendations by ACIP for the use of the Janssen COVID-19 vaccine for those over 18 years old [@doi:http://dx.doi.org/10.15585/mmwr.mm7009e4]. 
+The Janssen vaccine was first distributed to healthcare facilities on March 1st, 2021.
+While the CDC’s ACIP can provide recommendations, it is up to the public health authorities of each state, territory, and tribe to interpret the guidance and determine who will be vaccinated first [@url:https://www.hhs.gov/coronavirus/covid-19-vaccines/distribution/index.html].
+Prior to distribution of the Janssen vaccine, over 103 million doses of the Moderna and Pfizer-BioNTech vaccines were delivered across the U.S. with almost 79 million doses administered. 
+Of the total population, 15.6% have received at least one dose and 7.9% have received a second dose of either the Moderna (~38.3 million) or the Pfizer-BioNTech (~40.2 million) vaccines [@url:https://covid.cdc.gov/covid-data-tracker/#vaccinations].
+President Biden’s administration has predicted that by the end of May 2021 there may be enough vaccine supply available for all Adults in the U.S. [@url:https://www.cnn.com/2021/03/02/politics/biden-merck-johnson--johnson-vaccine/index.html; @url:https://www.bbc.com/news/world-us-canada-55721437].  
+However, vaccine production, approval, and distribution was not straightforward for U.S., as information was initially sparse and the rollout of vaccines was complicated by poor planning and leadership due to political activities prior to the change of administration in January 2021 [@url:https://ssrn.com/abstract=3770368]. 
```

Comment:
Well... it is just presenting the facts... I purposely didn't use the copious news sites as a reference here haha

I will let the decision to you. 

I also said political activities rather than name the specific individual... who I so desperately wanted to name...

Voldemort

Reply:
If someone has a nicer way to say these things, they can suggest it, haha. I agree with what you wrote!